identity-did/src/error.rs

```diff
@@ -50,17 +40,17 @@ pub enum Error {
   #[error("unable to modify relationships on embedded methods, use insert or remove instead")]
   InvalidMethodEmbedded,
 
+  /// Caused by attempting to revoke an unsupported method.
+  #[error("revocation is not supported for methods other than MerkleKeyCollection2021")]
```

Comment:
Good point, particularly since we're planning on replacing `MerkleKeyCollection2021` or adding an alternative. I think I'll add the `MethodType` to the variant to indicate on which method the revocation is _not_ supported.

Reply:
Changed to include `MethodType`.